tests/sles4sap/saptune/mr_test_run.pm

```diff
@@ -513,9 +513,10 @@ sub wrap_script_run {
         elsif ($output =~ /ASE.*notes.*1410736/) {
             record_info('Issue can be ignored, see jsc#TEAM-8662 for more details', "$output");
         }
-        elsif ($output =~ /transparent_hugepage.*always\s\[madvise\]\snever/) {
+        elsif ($output =~ /transparent_hugepage.*always\s\[madvise\]\snever ==/) {
```

Comment:
How the line this regexp is matching looks like?

Reply:
the line is possible like this:
```
transparent_hugepage         [FAIL]  always [madvise] never == always madvise [never]
transparent_hugepage         [FAIL]  always [madvise] never == [always] madvise never
transparent_hugepage         [FAIL]  [always] madvise never == always [madvise] never
```
but I didn't observe this issue happened, for exactly matching, so I added `==`, make sure to match the first one (it is the running status)

Reply:
For this output `transparent_hugepage         [FAIL]  [always] madvise never == always madvise [never]`  in job https://openqa.suse.de/tests/16061111#step/1_saptune_notes/99.  It cannot be handled by this regex.

Reply:
Hi @Amrysliu, the items about `transparent_hugepage` were removed in note files in new saptune, but not in note override files. so:
- the above issue will not happen because it happened to note files (not note override files).
- make sure the failed test is expected once the above issue `[always] madvise never == always madvise [never]` happened

Reply:
this issue https://openqa.suse.de/tests/16079822#step/1_saptune_overrides/818 will match the above regex and the test will pass.

Reply:
Understood. Thanks.